addons/notes/README.md

```diff
@@ -37,5 +37,8 @@ import { withNotes } from '@storybook/addon-notes';
 import Component from './Component';
 
 storiesOf('Component', module)
-  .add('with some emoji', withNotes({ notes: 'A very simple component'})(() => <Component></Component>));
+  .add('with some emoji', () => {
+    withNotes('some notes')
```

Comment:
@alexandrebodin minor nitpick, but this should have semicolon ... I think our linting is setup to fix this automatically and should be running before you can commit. I don't care too much about semiicolons, but am more concerned about bigger problems getting through